src/main/java/uk/gov/hmcts/probate/controller/DocumentController.java

```diff
@@ -233,4 +235,18 @@
         return ResponseEntity.ok(callbackResponseTransformer.addDocuments(callbackRequest,
                 Arrays.asList(document), null, null));
     }
+
+    @PostMapping(path = "/attach-scanned-docs-error", consumes = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<CallbackResponse> displayAttachScanDocErrorIfUsedFromUI(@RequestBody CallbackRequest callbackRequest) {
```

Comment:
take a look at the default exception handler class

Reply:
Updated with Sanjay's example in CaveatQueryService. Also changed the DefaultExceptionHandler to correct the order or messages.